src/components/app/list/expandedRow/expandedRow.css

```diff
@@ -15,16 +15,16 @@
 
 .expanded-row__close {
     border: none;
-    color: var(--white);
+    color: var(--N900);
```

Comment:
Can we accommodate these changes via formulae classes?

Reply:
Used few of them